save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/inputform/InputWithDebounce.kt

```diff
@@ -172,7 +172,6 @@ private fun <T> inputWithDebounce(
                 top = "100%".unsafeCast<Top>()
                 width = "100%".unsafeCast<Width>()
                 zIndex = "3".unsafeCast<ZIndex>()
-                overflow = "scroll".unsafeCast<Overflow>()
```

Comment:
Suggestion to make scrollable for `y` and forbid for `x`

Reply:
Fixed